ocaml/parsing/parsetree.mli

```diff
@@ -170,8 +170,8 @@ and core_type_desc =
 
 and arg_label = Asttypes.arg_label =
     Nolabel
-  | Labelled of string (** [label:T -> ...] *)
-  | Optional of string (** [?label:T -> ...] *)
+  | Labelled of string
```

Comment:
Why drop the comments?

Reply:
I was going to ask about this too. These comments already exist in `Asttypes`, so if I leave the ones here, the documentation is a little easier to find, but duplicated. Do we prefer to keep them?

Reply:
Oh, that makes sense. You can drop them.